airframe-msgpack/src/main/scala/wvlet/airframe/msgpack/spi/ReadCursor.scala

```diff
@@ -21,7 +21,9 @@ package wvlet.airframe.msgpack.spi
 case class ReadCursor(var buf: ReadBuffer, var position: Int) {
   private var offset: Int = 0
 
-  def lastReaadByteLength: Int = offset
+  @deprecated("Use #lastReadByteLength instead", since = "0.67")
+  def lastReaadByteLength: Int = lastReadByteLength
```

Comment:
airframe-msgpack has no internal usage right now. Just fixing the spelling is OK.